CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - SSHCollector as a configurable System info Collector. #1606
 - deployment_scrips/install-infection-monkey-service.sh to install an AppImage
   as a service. #1552
-- The ability to download the Monkey Island logs from the Infection Map page. #1640
+- The ability to download the Monkey Island logs from the Infection Map page. #1640, #1849
```

Comment:
Since this is a bug in a new, unreleased feature, we don't need to include the bugfix issue number. I'll drop this commit and merge.